src/apiCalls.js

```diff
@@ -0,0 +1,28 @@
+const getAllMovies = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+        .then(response => response.json())
+}
+
+const getMovieByID = (movieID) => {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}`)
+        .then(response => response.json())
+}
+
+// const getMovieByID = async (movieID) => {
+//     const movieDetails = await fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}`)
+//         return await movieDetails.json();
+//     // .then(response => response.json())
+// }
```

Comment:
Don't merge commented out code. If you don't need it, then you don't need it. This is just clutter.